frontend/src/components/VAudioTrack/VAudioTrack.vue

```diff
@@ -296,9 +296,9 @@ watch(
 
 /* Timekeeping */
 
-const message = computed(() =>
-  activeMediaStore.message
-    ? t(`audioTrack.messages.${activeMediaStore.message}`)
+const message = computed(() => 
+  activeMediaStore.id === props.audio.id && activeMediaStore.message 
```

Comment:
Remove trailing spaces. 

> [!TIP]
> This can be done automatically by setting up the pre-commit and pre-push Git hooks > or by running `ov just lint` manually.

```suggestion
const message = computed(() =>
  activeMediaStore.id === props.audio.id && activeMediaStore.message
```